### DISCUSSION TITLE
Github api to get user profile readme

### DISCUSSION THREAD

**Discussion by devadula-nandan** [community/community#60037] [Apps, API and Webhooks] [answered]
### Select Topic Area

Product Feedback

### Body

As the user profile readme is not associated with any repository, can we  get the content of user profile readme, in an api get method?

**Comment by dhilz** [comment]
use private acces token on seting and go to development options

**Comment by byrneh** [comment, marked as answer]
@devadula-nandan 

The user profile readme is asscoited with a reposirtory (details can be found here [#managing-your-profile-readme](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme) 

The special repositories readme can be retrieved with the contents API as below 
`curl -H "Authorization: Bearer $TOKEN --url https://api.github.com/repos/USERNAME/USERNAME/readme -H "Accept: application/vnd.github.raw"`

**Reply by devnyxie** [reply]
Thank you!
